ArchUnitNET/Domain/PropertyMember.cs

```diff
@@ -38,6 +39,8 @@ public class PropertyMember : IMember, ITypeInstance<IType>
 
         [CanBeNull] public MethodMember Setter { get; internal set; }
 
+        public bool IsInitSetter { get; }
```

Comment:
I would suggest "IsInitOnly" for consistency. Also, just "Init" might be taken to mean explicit initialization.